### PR TITLE
V0.7.0 doc updates

### PR DESCRIPTION
**Pull request by philippjfr**
Closes https://github.com/pyviz/panel/issues/754

### CODE REVIEW

File: examples/user_guide/Widgets.ipynb [hunk 1]
```diff
@@ -212,8 +214,9 @@
     "\n",
     "### Other\n",
     "\n",
-    "* **[``Audio``](../reference/widgets/Audio.ipynb)**: Displays a audio player given a local or remote audio file, providing access and control over the player state.\n",
-    "* **[``Button``](../reference/widgets/Button.ipynb)**: Allows triggering events when the button is clicked.  Unlike other widgets, it does not have a ``value`` parameter."
+    "* **[``Button``](../reference/widgets/Button.ipynb)**: Allows triggering events when the button is clicked.  Unlike other widgets, it does not have a ``value`` parameter.\n",
+    "* **[``Progress``](../reference/widgets/DataFrame.ipynb)**: A widget which allows displaying and editing a pandas DataFrame.\n",
```
Comment (jbednar) [inline examples/user_guide/Widgets.ipynb]:
```suggestion
    "* **[``Progress``](../reference/widgets/DataFrame.ipynb)**: A widget that allows displaying and editing a Pandas DataFrame.\n",
```

File: examples/getting_started/Introduction.ipynb [hunk 2]
```diff
@@ -207,22 +266,23 @@
...
+    "obj = RoomOccupancy()\n",
+    "obj"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `Sine` class and the `sine_obj` instance have no dependency on Panel, jupyter, or any other GUI or web toolkit; they simply declare facts about a certain domain (such as that sine waves take frequency and amplitude parameters, and that amplitude is a number greater than or equal to zero).  This information is then enough for Panel to create an editable and viewable representation for this object without having to specify anything that depends on the domain-specific details encapsulated in `sine_obj`:"
+    "The `RoomOccupancy` class and the `obj` instance have no dependency on Panel, Jupyter, or any other GUI or web toolkit; they simply declare facts about a certain domain (such as that sine waves take frequency and amplitude parameters, and that amplitude is a number greater than or equal to zero).  This information is then enough for Panel to create an editable and viewable representation for this object without having to specify anything that depends on the domain-specific details encapsulated in `obj`:"
```
[the remark this answers is not in the document]
Comment (jbednar) [reply examples/getting_started/Introduction.ipynb]:
```suggestion
    "The `RoomOccupancy` class and the `obj` instance have no dependency on Panel, Jupyter, or any other GUI or web toolkit; they simply declare facts about a certain domain (such as that smoothing requires window and sigma parameters, and that window is an integer greater than 0 and sigma is a positive real number).  This information is then enough for Panel to create an editable and viewable representation for this object without having to specify anything that depends on the domain-specific details encapsulated in `obj`:"
```

File: examples/user_guide/Widgets.ipynb (hunk 1 above)
Comment (philippjfr) [inline examples/user_guide/Widgets.ipynb]:
```suggestion
    "* **[``DataFrame``](../reference/widgets/DataFrame.ipynb)**: A widget that allows displaying and editing a Pandas DataFrame.\n",
```

File: doc/index.rst [hunk 3]
```diff
@@ -6,6 +6,19 @@
...
+.. raw:: html
+
+   <style>table {border-spacing: 15px} td { border: 1px solid black; vertical-align: top} </style>
+   <table>
+     <tr>
+       <td border=1><a href="https://anaconda.org/jbednar/datashadercliffordinteract"><b>Interact</b></a><br><a href="https://clifford.pyviz.demo.anaconda.com"><img src="_static/collage/interact.png" /></a></td>
+       <td border=1><a href="https://anaconda.org/jbednar/panel_gapminders"><b>Gapminders</b></a><br><a href="https://gapminder.pyviz.demo.anaconda.com"><img src="_static/collage/gapminders.png" /></a></td>
+       <td border=1><a href="https://anaconda.org/jbednar/nyc_taxi_panel"><b>NYC Taxi</b></a><br><a href="https://nyc-taxi.pyviz.demo.anaconda.com"><img src="_static/collage/nyc_taxi.png" /></a></td>
+       <td border=1><a href="https://anaconda.org/jbednar/glaciers"><b>Glaciers</b></a><br><a href="https://glaciers.pyviz.demo.anaconda.com"><img src="_static/collage/glaciers.png" /></a></td>
+       <td border=1><a href="https://anaconda.org/jbednar/eulersmethod"><b>Euler's Method</b></a><br><a href="https://anaconda.org/jbednar/eulersmethod"><img src="_static/collage/eulers_method.png" /></a></td>
```
[the remark this answers is not in the document]
Comment (philippjfr) [reply doc/index.rst]: -1, I really think the thumbnail should take them to the app.

File: examples/getting_started/Introduction.ipynb [hunk 4]
```diff
@@ -169,14 +228,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#sine_panel.show()"
+    "#occupancy.show()"
```
[the remark this answers is not in the document]
Comment (philippjfr) [reply examples/getting_started/Introduction.ipynb]: There's like three versions of the app in this notebook, not clear to me which to rename, whether to reuse names etc. I'll leave it for now.

File: doc/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (jbednar) [reply doc/index.rst]: I'm slightly uncomfortable going straight to the app, as it's out of Panel's control whether that app happens to be on a server that's running on any given time (as it's separate from the website), but sure, if you want to trust that it's up, going straight to the app is fine as long as we make sure apps include a link to their source.

In any case, none of these things should be pointing to my personal account; anything that goes to anaconda.org should be pointing to the examples.pyviz project instead.

[the remark this answers is not in the document]
Comment (philippjfr) [reply doc/index.rst]: Already fixed that.

**Pull request merged**